### DISCUSSION TITLE
i run on docker desktop fails

### DISCUSSION THREAD

**Discussion by b23618** [prisma/prisma#25190] [Q&A]
### Question

**### I build then run docker but to fails on Docker desktop and error can't connect database postgresql on RDS AWS**
error 'You may have to run ${Me("prisma generate")} for your changes to take effect.`,'
![2567-09-08 03_03_26-Window](https://github.com/user-attachments/assets/499e2948-c901-4a35-9233-1e5064b3980b)
![2567-09-08 03_03_43-Window](https://github.com/user-attachments/assets/2aebb8ac-eed6-4e69-a4c0-3923a0bfec1e)

How to fixed it ?

my Dockerfile

`FROM node:20 as build

WORKDIR /usr/app

COPY . .

RUN npm install && npm i -g prisma 

RUN prisma generate

RUN npm run build

EXPOSE 80

CMD ["node", "dist/main.js"]`

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
generator client {
  provider      = "prisma-client-js"
  binaryTargets = ["native"]
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: Windows
- Database: PostgreSQL
- Node.js version: 20

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @menphurk236 👋 

Thank you for raising this question.

Can you please confirm that you can access your database using a database GUI tool like [TablePlus](https://tableplus.com/) or [DBeaver](https://dbeaver.io/)? Is your rds instance publicly available? What does your `docker-compose.yml` file look like?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
